File: Linear_search/linear-search.java
import java.util.Scanner;
public class Main
{
	public static void main(String[] args) {
		Scanner sc=new Scanner(System.in);
		System.out.println("Enter number of elements of array");  
        int n =sc.nextInt();
        int[] array=new int[n];
        System.out.println("Enter elements of array"); 
        
        for (int i=0;i<n ;i++ ){
            array[i]=sc.nextInt();
        } 
        System.out.println("Enter value to search");  
        
        int element = sc.nextInt();  
        int i;
        for ( i = 0; i < n; i++){  
            if (array[i] == element){
                System.out.println("Element found at index: "+(i+1));  
                break;  
            }  
        }  
        if (i == n)
        System.out.println(element + " not found");  
	}
}

File: Linear_search/linear_search.c
#include <stdio.h>

int main()
{
  int array[100], search, c, n;

  printf("Enter the number of elements in array\n");
  scanf("%d", &n);

  printf("Enter %d integer(s)\n", n);

  for (c = 0; c < n; c++)
    scanf("%d", &array[c]);

  printf("Enter a number to search\n");
  scanf("%d", &search);

  for (c = 0; c < n; c++)
  {
    if (array[c] == search)    
    {
      printf("%d is present at index %d.\n", search, c+1);
      break;
    }
  }
  if (c == n)
    printf("%d is not present in the array.\n", search);

  return 0;
}
